regulations/templates/regulations/cfr_changes.html

```diff
@@ -1,9 +1,17 @@
 {% for instruction in instructions %}
-  <div style="border: 1px solid black;">{{ instruction }}</div>
+  <div class="cfr-changes">
```

Comment:
Quick note here: using a term like "instructions" might be more helpful -- "change" takes on a lot of different meanings in our code.